AtlaTeam/ojs
# Add svg arrow

Adds an SVG arrow icon for use with the PDF viewer plugin.

To Test:

* Open any PDF galley and confirm that the back arrow appears in the upper left corner.
